### PR TITLE
Add help page documenting weather visual animations

### PR DESCRIPTION
**Pull request by f-laurens**
Related to #16.
This is an initial step in providing embedded help for Pynab.
This PR adds a page that documents the weather visual animations and is linked from the _visual animation_ reference in the Weather widget.

### CODE REVIEW

**Comment by f-laurens** [comment]
At this point, **this PR is still work in progress**:
I'm not a Django expert and failed to find (in a finite amount of time ;-) a proper way to declare a new template (_dynamic_) Django page and reference it through a `href`...
I therefore created the page as a _static_ page: as such, it has (I hope) the appropriate look and feel, but it lacks I18N (no %trans tags possible).

This PR cannot therefore be merged as such: 
to finalize it, I need a good soul to provide me with help on own to properly declare the URL of a new template page to `nabweb`, and then reference it from the `nabweatherd` settings page.
@pguyot: any hint?
Then I will convert the static page to a dynamic page and add the proper localisation.

**Comment by f-laurens** [comment]
weather_animations

File: nabweb/static/nabweb/weather_animations.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <div class="card" id="nabweatherd-animations">
+          <div class="card-header">
+            <h5 class="card-title">The weather forecast visual animations</h5>
+          </div>
+
+          <div class="card-body">
+            <table>
+              <tr>
+                <td>Sunshine<br>Soleil</td>
+                <td><img src="images/soleil.gif"></td>
```
Comment (mehdichaouch) [inline nabweb/static/nabweb/weather_animations.html:37]: I suggest to use english files naming.

Comment (f-laurens) [reply nabweb/static/nabweb/weather_animations.html:37]: Nope. I kept the original file names from https://www.tagtagtag.fr/weather.html on purpose.

**Comment by f-laurens** [comment]
Static help page now converted to genuine template page (not sure I did it the 'right' way, but it works...), with localisation.
This PR is now ready for merge.

**Comment by f-laurens** [comment]
weather_help

**Comment by f-laurens** [comment]
Static help page now converted to genuine template page (not sure I did it the 'right' way, but it works...), with localisation.
This PR is **now ready for merge.**

File: nabweb/static/nabweb/weather_animations.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <td>Rain<br>Pluie</td>
+                <td><img src="images/pluie.gif"></td>
```
Comment (mehdichaouch) [inline nabweb/static/nabweb/weather_animations.html]: @f-laurens I see this PR is merged, but you didn't change files naming? That normal?

Comment (f-laurens) [reply nabweb/static/nabweb/weather_animations.html]: > you didn't change files naming? That normal?

Yes it is.  As already mentionned, I kept the original file names from https://www.tagtagtag.fr/weather.html on purpose.

**Pull request merged**